packages/pds/src/config.ts

```diff
@@ -120,6 +124,10 @@ export class ServerConfig {
     const emailNoReplyAddress =
       process.env.EMAIL_NO_REPLY_ADDRESS || 'noreply@blueskyweb.xyz'
 
+    const hiveApiKey = process.env.HIVE_API_KEY || undefined
```

Comment:
Once we land this, should things be all good if we run the pds without configuring a hive API key, or is it required?

Reply:
Oops, think I answered this for myself.

Reply:
yup if it's not included, it'll just fallback to keyword labeler